cl/assets/static-global/css/opinions.css

```diff
@@ -6,7 +6,7 @@
 .opinion-body {
 
   .harvard > * {
-    font-family: Merriweather, "Times New Roman", Times, serif;
+    font-family: "Source Sans Pro", Arial, Helvetica, sans-serif;
```

Comment:
Do you need this? Can't you just use the default?

Reply:
In theory no. but I personally find the default font to be off putting .

Reply:
Of course I am happy to do what you suggest.

Reply:
Not the time to discuss or think about fonts. Please remove.